packages/duckietown/include/duckietown/dtros/dtros.py

```diff
@@ -159,6 +163,17 @@ def __init__(self,
                 self.node_type,
                 health=self._health
             )
+
+        # Add a phase timer for profiling and a publisher topic
+        self._phase_timer = PhaseTimer()
+        self._pub_phase_timer = rospy.Publisher("~debug/phase_times", DiagnosticsPhaseTimingArray,
```

Comment:
this should be part of the `diagnostics` namespace, not `debug`

Reply:
done.